function-reference/install-functions/text.xml

```diff
@@ -431,6 +431,8 @@ The <c>*into</c> functions create the directory if it does not already exist.
       Normally executed to exclude from stripping.
       Eg. <c>dostrip -x /path/to/important.so</c>.  May also be used to include binaries
       to strip when <c>RESTRICT=strip</c> without the -x option.
+      Note that in order to recursively exclude directories from stripping, the absolute
+      path including <c>${ED}</c> must be provided, as opposed to a relative path.
```

Comment:
I don't think that's correct.
`dostrip` automatically prefixes its path with `${ED}`, see [PMS section 12.3.11](https://projects.gentoo.org/pms/8/pms.html#x1-13100012.3.11):

> Whether an item is to be excluded is determined as follows: For each item in the exclusion list, pretend it has the value of the ED variable prepended, then:
> - If it is a directory, act as if every file or directory immediately under this directory were in the exclusion list.
> - If the item is a file, the operation shall not be performed on it.
> - If the item does not exist, it is ignored.

Reply:
Oops, I just reread ionen's comment and looked at the ebuild in question I pulled this from, you are right.  What this was meant to say is that you must use an *absolute path relative to `${ED}`*.